src/App.tsx

```diff
@@ -1,8 +1,26 @@
-import React from 'react';
-import './App.css';
+import { useState, useEffect } from 'react';
+import './App.scss';
+import Search from './components/Search/index';
+import getSearch from './api/searchCities';
 
-function App() {
-    return <div className="App" />;
+function App(): JSX.Element {
+    const [search, setSearch] = useState<string>('');
+    const [items, setItems] = useState<string[]>([]);
+    useEffect(() => {
+        if (search) {
+            getSearch(search).then((res) => {
```

Comment:
with kind of search do you recieve here? is it a getter?
You should tell what the function is doing by giving the function name a verb as prefix.